packages/zipkin-transport-http/src/HttpLogger.js

```diff
@@ -9,7 +9,9 @@ class HttpLogger {
     const timer = setInterval(() => {
       this.processQueue();
     }, httpInterval);
-    timer.unref();
+    if (timer.unref) {
+      timer.unref();
```

Comment:
I think a comment is good just so we can point out why we are doing it. `timer.unref()` really only applies to Node where there's a running process that won't exit until the event loop is done. From a browser, there's still an event loop but the window doesn't close when the event loop is done processing events. Anyway, just one of those weird things about writing universal JS. 👍